documentation/guides/migrating-from-v3-to-v4.md

```diff
@@ -0,0 +1,390 @@
+# Migrating from v3 to v4
+
+Polaris v4 contains mostly internal changes and has simple migration path. This guide goes through the necessary steps to deal with breaking changes in your application.
+
+**Note:** This migration guide is for v3 but may apply for v2 as well.
```

Comment:
Can we include any info that would let the person know when the guide applies to v2? Just wondering how they would know if it applies

Reply:
We didn't have a lot of breaking changes in `v3` so a lot of this guide will apply. If you think this could be jarring, we could omit it?

Reply:
I think the title saying `Migrating from v3 to v4` clarifies what it's for. I'm mainly wondering if folks would want a link to more info about v2 so they can find out why we say "this guide may apply to v2" - I'd like them to have a link or more info so they can figure that out. Unless that's something they should already know?